osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -101,6 +103,11 @@ private void load(FrameworkConfigManager config, OsuConfigManager osuConfig, Gam
                     ItemSource = resolutions,
                     Current = sizeFullscreen
                 },
+                safeAreaConsiderationsCheckbox = new SettingsCheckbox
+                {
+                    LabelText = "Shrink game to avoid cameras and notches",
+                    Current = osuConfig.GetBindable<bool>(OsuSetting.SafeAreaConsiderations),
+                },
```

Comment:
The position of the checkbox feels weird, it's placed above more general options like "UI scaling" and "Screen scaling":

![IMG_3347](https://user-images.githubusercontent.com/22781491/198826596-ec14846e-951d-4c00-b40e-4f354b53ff86.jpg)

Think it would be better having it at the bottom instead?

Reply:
Disagree. This only shows on mobile. It makes sense for it to be the topmost setting for me.

Reply:
Perhaps the padding is what makes it look weird. Dunno.